museum/sampleRecords/adminDonor.destination.json

```diff
@@ -0,0 +1,1025 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.AccessionPrefix":"A",
+      "ca_objects.AccessionYear":"1988",
+      "ca_objects.AccessionNumber":"135",
+      "ca_objects.AccessionSeries":"0",
+      "ca_objects.AccessionBy":"jm",
+      "ca_entities":"UNKNOWN",
+      "ca_entities.entity_id":"1",
+      "ca_entities.preferred_labels.surname":"UNKNOWN",
+      "ca_entities.Phone":";;;;;;",
+      "ca_entities.MobilePhone":";;;;;;",
+      "ca_entities.Fax":";;;;;;",
+      "ca_entities.Email":";;;;;;",
+      "ca_entities.DonorNotes":";;;;;;",
+      "ca_entities.Deceased":";;;;;;",
+      "ca_entities.DonorAgentName":";;;;;;",
+      "ca_entities.DonorAgentAddress":";;;;;;",
+      "ca_entities.DonorAgentPhone":";;;;;;",
+      "ca_entities.LastEditBy":"urbaj",
+      "ca_entities.LastEditDate":"March 6 2015",
+      "ca_entities.LegacyID":"1896",
+      "ca_objects.acquisition_type_id":"Unknown",
+      "ca_objects.ConditionOnReceipt":"Good",
+      "ca_objects.LegalTitle.LegalTitleYesNo":"Yes",
+      "ca_objects.Copyright.CopyrightYesNo":"Yes"
+    },
+    {
+      "object_id":"2",
+      "id":"2",
+      "idno":"A1998.16",
+      "display_label":"GALLOP HOUSE",
+      "ca_objects.AccessionPrefix":"A",
+      "ca_objects.AccessionYear":"1998",
+      "ca_objects.AccessionNumber":"16",
+      "ca_objects.AccessionSeries":"0",
+      "ca_objects.AccessionBy":"jt",
+      "ca_entities":"UNKNOWN",
+      "ca_entities.entity_id":"1",
+      "ca_entities.preferred_labels.surname":"UNKNOWN",
+      "ca_entities.Phone":";;;;;;",
+      "ca_entities.MobilePhone":";;;;;;",
+      "ca_entities.Fax":";;;;;;",
+      "ca_entities.Email":";;;;;;",
+      "ca_entities.DonorNotes":";;;;;;",
+      "ca_entities.Deceased":";;;;;;",
+      "ca_entities.DonorAgentName":";;;;;;",
+      "ca_entities.DonorAgentAddress":";;;;;;",
+      "ca_entities.DonorAgentPhone":";;;;;;",
+      "ca_entities.LastEditBy":"urbaj",
+      "ca_entities.LastEditDate":"March 6 2015",
+      "ca_entities.LegacyID":"1896",
+      "ca_objects.acquisition_type_id":"Unknown",
+      "ca_objects.ConditionOnReceipt":"Good",
+      "ca_objects.LegalTitle.LegalTitleYesNo":"Yes",
+      "ca_objects.Copyright.CopyrightYesNo":"Yes"
+    },
+    {
+      "object_id":"3",
+      "id":"3",
+      "idno":"A1964.26a",
+      "display_label":"TOWN HALL PERTH, ETCHING",
+      "ca_objects.AccessionPrefix":"A",
+      "ca_objects.AccessionYear":"1964",
+      "ca_objects.AccessionNumber":"26",
+      "ca_objects.AccessionSeries":"0",
+      "ca_objects.AccessionParts":"a",
+      "ca_objects.AccessionBy":"jm",
+      "ca_entities":"UNKNOWN",
+      "ca_entities.entity_id":"1",
+      "ca_entities.preferred_labels.surname":"UNKNOWN",
+      "ca_entities.Phone":";;;;;;",
```

Comment:
Some of these fields have funky, incorrect looking values that are not coming from the source data.